### PR TITLE
admin: Add set config command

### PR DESCRIPTION
**Pull request by vadmeste**
Currently, it only work this way `cat config.json | mc admin config set myminio/`, 

Can we discuss if we need to add other way to pass the new config file ?

### CODE REVIEW

**Comment by harshavardhana** [comment]
> Currently, it only work this way cat config.json | mc admin config set myminio/,
> 
> Can we discuss if we need to add other way to pass the new config file ?

It behaves like `mc pipe`  without any piped input it reads from stdin so that things like this work 

```
mc admin config set myminio/ < config.json-mod
```

```
mc cat config.json-mod | mc config set myminio/
```

```
mc admin config set myminio/ 
{
	"version": "14",
	"credential": {
		"accessKey": "USWUXHGYZQYFYFFIT3RE",
		"secretKey": "MOJRH0mkL1IPauahWITSVvyDrQbEEIwljvmxdq03"
	},
	"region": "us-east-1",
....
```

File: cmd/admin-config-set.go [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  1. Set server configuration of a Minio server/cluster.
+     $ cat myconfig | {{.HelpName}} myminio/
+
+`,
+}
+
+// configSetMessage container to hold locks information.
+type configSetMessage struct {
+	Status               string               `json:"status"`
+	SetConfigStatus      bool                 `json:"-"`
```
Comment (harshavardhana) [inline cmd/admin-config-set.go]: If there is no plan to jsonify this you can keep this lower case.

File: cmd/admin-config-set.go [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+	defer func() {
+		msg = console.Colorize("SetConfig", msg)
+	}()
+	if u.SetConfigStatus {
+		msg += "Setting new configuration file has been successful.\n"
+	} else {
+		msg += "Setting new configuration file has failed.\n"
+	}
+	for _, s := range u.SetConfigNodeSummary {
+		msg += fmt.Sprintf("  %s.. ", s.Name)
```
Comment (harshavardhana) [inline cmd/admin-config-set.go]: Can you paste a sample output how this looks ?

Comment (vadmeste) [inline cmd/admin-config-set.go]:
```
$ mc admin config set myminiold1/ < config.json
Setting new configuration file has been successful.
  172.17.0.2:9000.. OK.
  172.17.0.3:9000.. OK.
  172.17.0.4:9000.. OK.
  172.17.0.5:9000.. OK.
```

Comment (harshavardhana) [inline cmd/admin-config-set.go]: Actually here the output would be nicer to be like - success is printed at the end based on the errors..

```
$ mc admin config set myminiold1/ < config.json
New configuration applied to `172.17.0.2:9000`  successfully.
New configuration applied to `172.17.0.3:9000`  successfully.
New configuration applied to `172.17.0.4:9000`  successfully.
New configuration applied to `172.17.0.5:9000`  successfully.
Setting new Minio configuration has been successful.
```

Comment (vadmeste) [inline cmd/admin-config-set.go]: Yeah it is nicer like that.

**Pull request merged**